timeless/reservations/views.py

```diff
@@ -36,14 +38,15 @@ def delete(self, id):
         return self.ctr.delete_reservation_settings(id), HTTPStatus.NO_CONTENT
 
 
-class CommentView(CrudAPIView):
+class CommentView(CrudAPIView, SecuredView):
```

Comment:
@amihaiemil `dispatch_request` method of `SecuredView` will be never executed, since python will search the method in `CrudAPIView` and then in its base class - `views.MethodView`, but method view doesn't call `super`. You know, tricks of multiple inheritance in python. So the right code is:
```python
class CommentView(SecuredView, CrudAPIView):
```

Ideally `SecuredView`:
1. should be called `SecuredMixin`
2. should implement only `dispatch_request` and check permissions
3. should not be inherited from `views.MethodView`

But it's a different story. Let's start with changing ordering of base classes for `CommentView``